test/primitive-tests.cabal

```diff
@@ -31,6 +31,12 @@ test-suite test
   build-depends: base >= 4.5 && < 4.12
                , ghc-prim
                , primitive
+               , QuickCheck
+               , tasty
+               , tasty-quickcheck
+               , tagged
+               , transformers >= 0.3
+               , quickcheck-classes == 0.4.2
```

Comment:
To make this more robust in the future, let's do `>= 0.4.2` instead.

Reply:
I'm a little worried about accidentally break `primitive`'s test suite with a future release of `quickcheck-classes`, but I'll go ahead and do this.

Reply:
If that is a concern, it might be worthwhile to add an upper version bound of `< 0.5`.